sway/commands/resize.c

```diff
@@ -457,7 +456,7 @@ static struct cmd_results *cmd_resize_set(int argc, char **argv) {
 		if (argc > num_consumed_args) {
 			return cmd_results_new(CMD_INVALID, "%s", usage);
 		}
-		if (width.unit == MOVEMENT_UNIT_INVALID) {
+		if (height.unit == MOVEMENT_UNIT_INVALID) {
```

Comment:
Good catch! This as a bugfix (without the error handling changes below) looks good to me.